src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Unix.cs

```diff
@@ -335,20 +335,19 @@ internal void LogBuffer(int size)
         private SocketError FinishOperationAccept(Internals.SocketAddress remoteSocketAddress)
         {
             System.Buffer.BlockCopy(_acceptBuffer!, 0, remoteSocketAddress.Buffer, 0, _acceptAddressBufferCount);
-            Socket? sukru = _acceptSocket;
+            Socket? tempAcceptSocket = _acceptSocket;
             _acceptSocket = _currentSocket!.CreateAcceptSocket(
                 SocketPal.CreateSocket(_acceptedFileDescriptor),
                 _currentSocket._rightEndPoint!.Create(remoteSocketAddress));
-            if (sukru != null)
+            if (tempAcceptSocket != null)
             {
-                sukru.DisposeHandle();
-                sukru.CopyStateFromSource(_acceptSocket);
+                tempAcceptSocket.DisposeHandle();
+                tempAcceptSocket.CopyStateFromSource(_acceptSocket);
                 // We keep this socket to make clean-up.
```

Comment:
I don't understand what this comment is saying.

Reply:
I was trying to say, "let's save this temp socket for now and clean it later.", however, I'm going to change the implementation as you proposed, so this comment will be gone.